fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIORoot.scala

```diff
@@ -34,14 +34,16 @@ sealed trait BIORootInstancesLowPriority1 extends BIORootInstancesLowPriority2 {
 }
 
 sealed trait BIORootInstancesLowPriority2 extends BIORootInstancesLowPriority3 {
-  @inline implicit final def BIOConvertFromBIOProfunctor[FR[-_, +_, +_]](implicit BIOProfunctor: NotPredefined.Of[BIOProfunctor[FR]]): BIOFunctor3[FR] with S3 = S3(BIOProfunctor.InnerF)
+  @inline implicit final def BIOConvertFromBIOParallel[FR[-_, +_, +_]](implicit BIOParallel: NotPredefined.Of[BIOParallel3[FR]]): BIOMonad3[FR] with S5 = S5(BIOParallel.InnerF)
 
   @inline implicit final def AttachBIOProfunctor[FR[-_, +_, +_]](@deprecated("unused", "") self: BIOFunctor3[FR])(implicit BIOProfunctor: BIOProfunctor[FR]): BIOProfunctor.type = BIOProfunctor
   @inline implicit final def AttachBIOAsk[FR[-_, +_, +_], R](@deprecated("unused", "") self: BIOFunctor3[FR])(implicit BIOAsk: BIOAsk[FR]): BIOAsk.type = BIOAsk
   @inline implicit final def AttachBIOBifunctor[FR[-_, +_, +_], R](@deprecated("unused", "") self: BIOFunctor3[FR])(implicit BIOBifunctor: BIOBifunctor3[FR]): BIOBifunctor.type = BIOBifunctor
+  @inline implicit final def AttachBIOParallel[FR[-_, +_, +_], R](@deprecated("unused", "") self: BIOMonad3[FR])(implicit BIOParallel: BIOParallel3[FR]): BIOParallel.type = BIOParallel
 }
 
 sealed trait BIORootInstancesLowPriority3 extends BIORootInstancesLowPriority4 {
+  @inline implicit final def BIOConvertFromBIOProfunctor[FR[-_, +_, +_]](implicit BIOProfunctor: NotPredefined.Of[BIOProfunctor[FR]]): BIOFunctor3[FR] with S3 = S3(BIOProfunctor.InnerF)
   @inline implicit final def BIOConvertFromBIOBifunctor[FR[-_, +_, +_]](implicit BIOBifunctor: NotPredefined.Of[BIOBifunctor3[FR]]): BIOFunctor3[FR] with S4 = S4(BIOBifunctor.InnerF)
```

Comment:
@Adriani277 Now you'll have to move Bifunctor down to priority 4 - otherwise if both context bounds are present - they will conflict, BIZIO down to priority 5 & BIOConvert3To2 down to priority 6
i.e. shift everything down by one

Reply:
Ahh I see, will do that now

Reply:
I have performed the changes but I noticed that moving `BIOConvert3to2` to priority 6 causes an error in `BIORoot`
```
could not find implicit value for parameter ev: izumi.functional.bio.BIOFunctor[zio.IO]
[E]      L23:       Semaphore[Task](permits)(catz.BIOAsyncForkToConcurrent[IO])
```

I am trying to find a fix but no luck so far